docs/scipp_widgets_intro.ipynb

```diff
@@ -0,0 +1,184 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Scipp widgets examples\n",
+    "\n",
+    "The scipp-widgets module contains some simple wrapper functions to aid in creating graphical notebooks. To do this it builds upon the [ipywidgets](https://ipywidgets.readthedocs.io/) module.\n",
+    "\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
```

Comment:
I think you could do with a better introduction in this notebook. Not entirely clear what problem is being solved then on the next cell a whole bunch of imports. Maybe combine the imports with the cell below that uses them?